src/sentry/integrations/github/client.py

```diff
@@ -169,10 +178,10 @@ def get_trees_for_org(self, gh_org: str, cache_seconds: int = 3600 * 24) -> Dict
         extra = {"gh_org": gh_org}
         try:
             repositories = self._populate_repositories(gh_org, cache_seconds)
```

Comment:
I'm not putting this method behind API rate limit protection since it requests 100 repositories per page. An org with 4,800 repositories would only use 48 requests for this.

I can add this support in a following PR if we believe we still need it.